src/components/topic/partials/topic.html

```diff
@@ -8,12 +8,12 @@ <h4 class="modal-title" translate>choose_theme</h4>
       <div class="modal-body">
         <div class="input-modal-container">
           <div class="ga-topic-item"
-               ng-repeat="topic in ::topics"
+               ng-repeat="topic in translateTopics(topics) | orderBy:['groupId', 'name']"
```

Comment:
Same comment. Using `translateTopics` here you will translate the topics on every digest cycle and not only when the language change.